connect/go.mod

```diff
@@ -7,3 +7,5 @@ require (
 	github.com/spf13/cobra v1.1.1
 	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad
 )
+
+replace github.com/abrander/garmin-connect => ../
```

Comment:
I think this was left by mistake..? But would it make sense to update `go.mod` and `go.sum` now that we're at it?

Reply:
Maybe I should just remove this, I'm not entirely sure what it should be and haven't had a chance to look properly.

Reply:
I believe you added the line to assist when developing from your own clone, and not `github.com/abrander/garmin-connect`.

Reply:
Removed the change!